lib/usage.js

```diff
@@ -173,12 +173,12 @@ module.exports = function (yargs, y18n) {
         if (~options.array.indexOf(key)) type = '[' + __('array') + ']'
 
         var extra = [
-            type,
-            demanded[key] ? '[' + __('required') + ']' : null,
-            options.choices && options.choices[key] ? '[' + __('choices:') + ' ' +
-              self.stringifiedValues(options.choices[key]) + ']' : null,
-            defaultString(options.default[key], options.defaultDescription[key])
-          ].filter(Boolean).join(' ')
+          type,
+          demanded[key] ? '[' + __('required') + ']' : null,
+          options.choices && options.choices[key] ? '[' + __('choices:') + ' ' +
+            self.stringifiedValues(options.choices[key]) + ']' : null,
+          defaultString(options.default[key], options.defaultDescription[key])
+        ].filter(Boolean).join(' ')
```

Comment:
Change is out-of-scope, PR denied.

j/k

Sorry, not sure if comments for the sake of humor are allowed yet. :sunglasses: